src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,48 @@
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestState: function (id, url) {
+    JSONPUtil.request(`${url}/state`).then(
+      function (state) {
+        AppDispatcher.dispatch({
+          actionType: MesosEvents.REQUEST_STATE_COMPLETE,
+          data: {id: id, state: state}
+        });
+      },
+      function (error) {
+        AppDispatcher.dispatch({
+          actionType: MesosEvents.REQUEST_STATE_ERROR,
+          data: error
+        });
+      }
+    );
+  },
+  requestFiles: function (id, host, filePath) {
+    JSONPUtil.request(
+      `${host}/files/browse?path=${encodeURIComponent(filePath)}`)
```

Comment:
OK ```/browse``` doesn't work with Mesos 0.24, it must be ```/browse.json```. Then it works!
Could you test if browse.json is also working with 0.25. Then everything is perfect.